src/torrent/data/file_list.cc

```diff
@@ -160,10 +160,10 @@ uint64_t
 FileList::free_diskspace() const {
   uint64_t freeDiskspace = std::numeric_limits<uint64_t>::max();
 
-  for (path_list::const_iterator itr = m_indirectLinks.begin(), last = m_indirectLinks.end(); itr != last; ++itr) {
+  for (const auto& m_indirectLink : m_indirectLinks) {
     rak::fs_stat stat;
 
-    if (!stat.update(*itr))
+    if (!stat.update(m_indirectLink))
```

Comment:
Don't use `m_` prefix, call it just `link`.

Reply:
clang-tidy did this. Fixed.